modules/vaos/app/controllers/vaos/preferences_controller.rb

```diff
@@ -0,0 +1,18 @@
+# frozen_string_literal: true
+
+require_dependency 'vaos/application_controller'
+
+module VAOS
+  class PreferencesController < ApplicationController
```

Comment:
here it's plural... a user can only have 1 right? Maybe it needs to be a singular resource?

Show instead of Index?

Reply:
If we change it to be show, get single preference, what do we use for the id? I think we'd have to make a custom route.

Reply:
oh the preference also has an id? Shouldn't the ID just be the patient identifier / icn??

Reply:
i think i'm ok with index, since "preferences" reads better anyway. Lets just make it consistent maybe.